conf/modules.config

```diff
@@ -71,6 +71,14 @@ process {
                 path: { "${params.outdir}/fastp" },
                 mode: params.publish_dir_mode,
                 pattern: '*.{log,html,json}'
+            ],
+            [
+                path: { "${params.outdir}/analysis_ready_fastqs" },
+                mode: params.publish_dir_mode,
+                pattern: '*.fastq.gz',
+                enabled: params.save_analysis_ready_fastqs,
+                // Don't know why `!` doesn't work here, but `== false` makes it work...
+                saveAs: { ( params.perform_runmerging == false || ( params.perform_runmerging && !meta.is_multirun ) ) && !params.perform_shortread_hostremoval && !params.perform_shortread_complexityfilter && params.perform_shortread_qc && params.save_analysis_ready_fastqs ? it : null }
```

Comment:
😱

Reply:
I know I know... I'll refactor in the future™️